### ISSUE TITLE
there is something wrong with Toggle current app in black list

### ISSUE THREAD

**Issue by FengweiHu** [Toxblh/MTMR#120]
I think there is something wrong with Toggle current app in black list in the  new version 0.18.5, pls make a check, it is different with the former versions, thanks.

**Comment by Toxblh** [comment]
What do you mean?
So as a solution, soon I add the current toggle status for blacklist like Hide or Start at Login

**Comment by FengweiHu** [comment]
ok this solution is good.

**Comment by FengweiHu** [comment]
Ok, now I toggle safari into the black list, the question is if I move to other apps or desktop from safari, MTMR will also hide, it seems like the other apps in the black list too but actually not.

**Comment by Toxblh** [comment]
qq: Do you use the button with type `exitTouchbar` on MTMR?

**Comment by FengweiHu** [comment]
no, I use escape instead of exit.

<img width="724" alt="2018-10-18 17 09 17" src="https://user-images.githubusercontent.com/4504959/47143810-99de3880-d2f8-11e8-9339-2d33b3ae7212.png">

**Comment by FengweiHu** [comment]
All right let me try again.

**Comment by Toxblh** [comment]
https://streamable.com/jebhw

**Comment by Toxblh** [comment]
I have weChat /toxblh

**Comment by FengweiHu** [comment]
ok add you WeChat, pls confirm it.

**Comment by FengweiHu** [comment]
> https://streamable.com/jebhw

ok I see it, I will send you a video with WeChat
